code/modules/cm_preds/yaut_weapons.dm

```diff
@@ -230,7 +230,7 @@
 	item_state = "scythe_double"
 
 //Combistick
-/obj/item/weapon/yautja/combistick
+/obj/item/weapon/yautja/combistick/real
```

Comment:
```suggestion
/obj/item/weapon/yautja/chained/combistick
```
Repath all this to /obj/item/weapon/yautja/chained/ so the typepath is indicative of what the weapon actually is, as the axe is not a combistick at all, it's just copying the code to be attached to the pred by a chain.

Reply:
Completely agree. working on it now

Reply:
repathed it all, retested and I've noticed no issues. Thanks forest